### ISSUE TITLE
Update to python 3.8

### ISSUE THREAD

**Issue by eugenesvk** [SublimeText/Pywin32#10]
I've copy&pasted a few folders (except for demos and tests) from PyPi's 3.8 wheels to this [fork](https://github.com/eugenesvk/Pywin32) , but not sure whether that's the proper way

**Comment by FichteFoll** [comment]
Ideally, we'd update the pywin32 entry at https://github.com/packagecontrol/channel to point to the [whl files on pypi directly](https://pypi.org/project/pywin32/#files).

**Comment by FichteFoll** [comment]
Never mind, they already do. https://github.com/packagecontrol/channel/blob/8da3e2a0996af2fa30f776ba7854346d51867986/repository.json#L968-L973

**Comment by eugenesvk** [comment]
Nice, but why is this not available on package control website? Is this the new package control that's only planned?

**Comment by FichteFoll** [comment]
Ah yes, I see now. This is the old package repo that has since been deprecated by the availability of a library, which is the better use case for Python dependencies like this one. I should update the readme with this.

TLDR: don't use this package, use the library via https://packagecontrol.io/docs/dependencies.

**Comment by FichteFoll** [comment]
I have updated the readme to mention this. If you are having problems with the library, I suggest to hop into our discord (mentioned at https://github.com/SublimeText) and see if there are other people that might be able to assist.
